plugins/defaultplugins/aclplugin/acl_config.go

```diff
@@ -111,7 +111,7 @@ func (plugin *ACLConfigurator) ConfigureACL(acl *acl.AccessLists_Acl) error {
 
 	rules, isL2MacIP := plugin.validateRules(acl.AclName, acl.Rules)
 	// Configure ACL rules.
-	var vppACLIndex uint32
+	var vppACLIndex int32
```

Comment:
I wonder why is it changed to int32, if you have to cast it everywhere. Also methods 'AddMacIPAcl' and 'AddIPAcl' return int32 just because it is casted from uint32...